core/src/main/scala/kafka/server/ClientQuotaManager.scala

```diff
@@ -115,7 +116,7 @@ class ClientQuotaManager(private val config: ClientQuotaManagerConfig,
   private val lock = new ReentrantReadWriteLock()
   private val delayQueue = new DelayQueue[ThrottledResponse]()
   private val sensorAccessor = new SensorAccess(lock, metrics)
-  val throttledRequestReaper = new ThrottledRequestReaper(delayQueue)
+  val throttledRequestReaper = new ThrottledRequestReaper(delayQueue, threadNamePrefix.getOrElse(""))
```

Comment:
One very minor comment: this could be private[kafka.server] as we're using it only from ClientQuotaManager and ClientQuotaManagerTest as I see.

Reply:
@viktorsomogyi Thank you for the review, updated.